app/app/src/main/java/com/example/clubwat/views/ClubDetailsView.kt

```diff
@@ -189,6 +188,11 @@ fun ClubDetailsView(
                     )
                     if (club != null) {
                         LazyColumn {
+                            if (club!!.isClubAdmin) {
```

Comment:
This should be in club management view. Copy what was done with user management

Reply:
resolved.